### ISSUE TITLE
VSCode Highlights those lines in red even though they're valid configuration.

### ISSUE THREAD

**Issue by ikhal3d** [codeout/vscode-junos#22]
Hi,

As the subjet title states, the following are valif configuration, yet the extension flags them in red indicating there is syntax errors.

Cheers.

```
set chassis cluster reth-count 2
#
set chassis cluster redundancy-group 1 node 0 priority 100
set chassis cluster redundancy-group 1 node 1 priority 50
#
set security log stream SYSLOG_SERVER transport protocol tcp
#
set forwarding-options dhcp-relay server-group DHCP_SERVER 192.168.0.1
```

**Comment by codeout** [comment]
Hi,

Thank you for your reporting. I could reproduce the issue.

There are two types of problems,
- missing SRX-specific syntax
- junoser bug

Let me look into both of them.

**Comment by codeout** [comment]
Hi @ikhal3d,

I've just published v0.3.1 with the fixes.

<img width="962" alt="image" src="https://github.com/codeout/vscode-junos/assets/921696/3ed92b95-78c8-4939-a9ab-f9de53ce2b2d">

Please reopen this issue if you see any further problems.
